Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Validator.rst

```diff
@@ -67,12 +71,28 @@ The validator is used as annotation in the action methods of the controller:
 ..  literalinclude:: _CustomValidator/_ObjectValidatorUsage.php
     :caption: EXT:blog_example/Classes/Controller/BlogController.php, modified
 
+..  _extbase_domain_validator-di:
+
 Dependency injection in validators
 ==================================
 
-Starting with TYPO3 v12 Extbase validators are capable of :ref:`dependency injection <Dependency-Injection>`
+Extbase validators are capable of :ref:`dependency injection <Dependency-Injection>`
 without further configuration, you can use the constructor method:
 
 ..  literalinclude:: _Validator/_MyCustomValidator.php
-    :language: php
+    :caption: EXT:my_extension/Classes/Validators/MyCustomValidator.php
+
+..  _extbase_domain_validator-request:
+
+Request object in Extbase validators
+====================================
+
+..  versionadded:: 13.2
+    Extbase :php-short:`\TYPO3\CMS\Extbase\Validation\Validator\AbstractValidator`
+    provides a getter and a setter for the PSR-7 Request object.
+
+You can use the PSR-7 request object in a validator, for example to get
+the site settings:
+
```

Comment:
You already have written that at the very top

Reply:
Not really,  the version added information is in a box and will be removed in  the future. The second sentence is the introduction of the Example and will stay.